wp-seo-main.php

```diff
@@ -11,7 +11,7 @@
 }
 
 /**
- * @internal Nobody should be able to overrule the real version number as this can cause serious issues
+ * @note Nobody should be able to overrule the real version number as this can cause serious issues
```

Comment:
@internal is a PHPDoc tag, @note is not. @internal is used to indicate the info is for developers only
Ref: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/internal.html